azure-pipelines/code-coverage/common.yml

```diff
@@ -38,7 +38,7 @@ jobs:
   - task: Gradle@2
     displayName: Run common code coverage
     inputs:
-      tasks: common:localDebugCommonUnitTestCoverageReport -Plabtest -ProbolectricSdkVersion=${{variables.robolectricSdkVersion}}
```

Comment:
why remove `-Plabtest`?

Reply:
we should know about lab tests in coverage reports

Reply:
See https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/1554#discussion_r758464826